### PR TITLE
rustdoc: improve stability mark arrows

### PR DESCRIPTION
**Pull request by liigo**
### current

old-stability-arrow

### new

new-stability-arrow

### new dark

dark-stability-arrow

### CODE REVIEW

**Comment by liigo** [comment]
cc https://github.com/rust-lang/rust/issues/51387

File: src/librustdoc/html/static/rustdoc.css [hunk 1]
```diff
@@ -542,11 +542,13 @@ h4 > code, h3 > code, .invisible > code {
...
-	font-size: 30px;
+	content: '⤶';
+	transform:rotate(90deg);
+	font-size: 20px;
 	position: absolute;
-	top: -9px;
-	left: -13px;
+	top: -6px;
+	left: -15px;
+	color: #999;
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/rustdoc.css]: Everything color-linked should be put inside light.css and dark.css files (even if it's the same in both files.

**Review by GuillaumeGomez — CHANGES_REQUESTED** [review]
Just one change to make, otherwise looks good to me.

**Comment by liigo** [comment]
@GuillaumeGomez  done.

File: src/librustdoc/html/static/rustdoc.css [hunk 2]
```diff
@@ -542,11 +542,12 @@ h4 > code, h3 > code, .invisible > code {
 }
 
 .content .stability::before {
-	content: '˪';
-	font-size: 30px;
+	content: '⤶';
+	transform:rotate(90deg);
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/rustdoc.css]: Also I'm not a fan of this. I'm pretty sure the expected character exists as is.

Comment (liigo) [reply src/librustdoc/html/static/rustdoc.css]: I failed to find it. Will check that again.

Comment (liigo) [reply src/librustdoc/html/static/rustdoc.css]: changed to another arrow: ⬑

rotate-vs-nonrotate

**Comment by liigo** [comment]
Now we use the 'Leftwards Arrow with Tip Upwards': ⬑ (U+2B11) ~~(Not the rotated ⤶)~~

rotate-vs-nonrotate

**Comment by GuillaumeGomez** [comment]
Thanks a lot!

@bors: r+ rollup

**Pull request merged**